Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
sorry. I was not talking about pyenv, but rather pipenv

basically with pipenv you do
```
sudo pip install pipenv  # install once
pipenv install --three  # or pipenv install --two
```
pipenv requires a Pipfile created, which describe the list of dependencies needed for prod and development